src/rest_framework_dso/views.py

```diff
@@ -41,12 +41,15 @@ def get_view_name(view):
     if name == "Schema":
         name = getattr(view.schema_generator, "title", "Schema")
 
-    # Add the title for instances
+    # Use the title for instances
     response = getattr(view, "response", None)
     suffix = getattr(view, "suffix", None)
 
     if suffix == "Instance" and response:
-        name = response.data["_links"]["self"]["title"]
+        if response.status_code == 200:
```

Comment:
Nice solution!

Reply:
thanks

Reply:
Please use `rest_framework.status. HTTP_200_OK`. Also adding test is good idea.